geode-assembly/src/test/java/org/apache/geode/rest/internal/web/RestSecurityIntegrationTest.java

```diff
@@ -94,7 +94,7 @@ public static void before() throws Exception {
   @Test
   public void testFunctions() throws Exception {
     String json = "{\"@type\":\"double\",\"@value\":210}";
-
+    Thread.sleep(500000);
```

Comment:
This seems like a really long pause for an integration test.  Is there another way to do this?